src/server/GraphQLServer.ts

```diff
@@ -269,11 +293,16 @@ export class GraphQLServer {
             documentAST = this.parseFunction(new Source(requestInformation.query,
                 'GraphQL request'))
         } catch (syntaxError: unknown) {
+            this.logger.error(requestCouldNotBeProcessed,
+                syntaxError as GraphQLError,
```

Comment:
This may be correct, but I wonder why would you catch "unknown" only to cast it to a GraphQLError? Is it not possible to to something like this:
`} catch (syntaxError: GraphQLError) {`

Reply:
In the Typescript version we are using this seems not to be possible. I'm not sure if this might get fixed in future Typescript versions but for now it is only possible to use any or unknown in catch clause.
See: "TS1196: Catch clause variable type annotation must be 'any' or 'unknown' if specified."

Reply:
Ok, thanks for explaining.